app/controllers/passengers_controller.rb

```diff
@@ -0,0 +1,67 @@
+class PassengersController < ApplicationController
+  def index
+    @passengers = Passenger.all
+  end
+
+  def show
+    @passenger = Passenger.find_by(id: params[:id])
+    if @passenger.nil?
+      head :not_found
+      return
+    end
+  end
+
+  def new
+    @passenger = Passenger.new
+  end
+
+  def create
+    @passenger = Passenger.new(passenger_params)
+    if @passenger.save
+      redirect_to passenger_path(@passenger.id)
+      return
+    else
+      render :new
```

Comment:
You should include a custom status here:

```suggestion
      render :new, status: :bad_request
```